docs/guides/layout-spacing.md

```diff
@@ -0,0 +1,92 @@
+---
+title: Layout Spacing
+category: Guides
+order: 8
+---
+
+# Layout Spacing
+
+Our design system provides a set of spacing tokens for consistent layouts and components. Some tokens share values but should be used semantically. For instance, while both `space12` and `buttons` are 12px, `buttons` should be used for spacing between buttons.
+
```

Comment:
we should mention the legacy tokens here too

Reply:
good point, I'll do that